test_util/installer_api_test.py

```diff
@@ -343,3 +343,6 @@ def deploy(self, expect_errors=False):
 
     def postflight(self, expect_errors=False):
         self.run_cli_cmd('--postflight', expect_errors=expect_errors)
+
+    def uninstall(self, expect_errors=False):
```

Comment:
the dgc.sh --uninstall is actually something we need to kill or rework.  `pkgpanda uninstall` is the canonical we should test.